CHANGELOG.md

```diff
@@ -33,12 +32,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - **BREAKING**: `PdfLoader` no longer accepts `str` file content, `Path` file paths or `IO` objects as sources. Instead, it will only accept the content of the PDF file as a `bytes` object.
 - **BREAKING**: `TextLoader` no longer accepts `Path` file paths as a source. It will now accept the content of the text file as a `str` or `bytes` object.
 - **BREAKING**: `FileManager.default_loader` is now `None` by default.
-- **BREAKING**: Replaced `EventListener.handler` with `EventListener.driver` and `LocalEventListenerDriver`.
 - **BREAKING** Bumped `pinecone` from `^2` to `^3`.
```

Comment:
are there migration steps customers need to take, or did we cover it all?

Reply:
There would've been had we kept this breaking change. But now that I've reverted it, users will just use Event Listeners as they do today.